hack/test/clusterdeploy.yaml

```diff
@@ -14,69 +14,40 @@ metadata:
   namespace: certtest
 spec:
   installed: true
-  baseDomain: dnstest.aws.openshift.com
+  baseDomain: dev.yuggoth.space
```

Comment:
Maybe make generic like app.example.com?

Reply:
Can we move this file into something like hack/test/deploy?

Reply:
ugh, i thought i excluded that change. i'll revert hostname change and move the clusterdeployment into hack/test/deploy.yml